test/functional/assumevalid.py

```diff
@@ -54,16 +54,16 @@ def send_header_for_blocks(self, new_blocks):
         self.send_message(headers_message)
 
 class AssumeValidTest(BitcoinTestFramework):
-    def __init__(self):
-        super().__init__()
+    def set_test_params(self):
         self.setup_clean_chain = True
```

Comment:
It seems there are more `setup_clean_chain = True` so why not make it the default?

Reply:
I haven't made any changes to `setup_clean_chain`. See earlier comment:

> I haven't done the same for self.setup_clean_chain, since I think it makes more sense for that to have a default. If people feel strongly, it can be added in a future commit.

Feel free to open a PR to change/remove the default if you think that's necessary. I don't think it's required for this PR.